tests/perf-system/benchmarks/benchmarks.py

```diff
@@ -0,0 +1,513 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+
+import json
+import os
+import random
+import signal
+import stat
+import string
+import subprocess
+import sys
+import time
+from pathlib import Path
+from loguru import logger as LOG
+
+import paramiko
+
+
+def connect_to_remote_server(ip_address):
+    client = paramiko.SSHClient()
+    client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+    client.connect(ip_address)
+    return client
+
+
+def clean_up_cchost(nodes):
+    LOG.info("Cleaning up previous CCF processes")
+    for node in set([address.split(":")[0] for address in nodes]):
+        client = connect_to_remote_server(node)
+        LOG.info(f"Killing cchost on {node}")
+        client.exec_command("killall cchost")
+        client.close()
+
+
+# TODO: make recursive, copies only first two layers
+def fetch_remote_dir(ip_address, remote_dir: Path, local_dir: Path):
+    client = connect_to_remote_server(ip_address)
```

Comment:
pysftp seems to support this is and looks fairly easy to use:

https://pysftp.readthedocs.io/en/latest/cookbook.html#pysftp-connection-get-r
https://pysftp.readthedocs.io/en/latest/index.html

Reply:
done in d8be6a62b0b614866de8c6eab86235d2aa697194